alcs-frontend/src/app/features/search/file-type-filter-drop-down/file-type-filter-drop-down.component.ts

```diff
@@ -173,7 +173,7 @@ export class FileTypeFilterDropDownComponent implements AfterViewInit {
   }
 
   ngAfterViewInit(): void {
-    this.treeControl.expandAll();
+    //this.treeControl.expandAll();
```

Comment:
I feel like we might as well just clean up this whole function.

Reply:
Removed